operations/use-cf-services.yml

```diff
@@ -48,57 +43,13 @@
   path: /variables/name=postgres_client/options/alternative_names/-
   value: ((deployment_name))-postgres.tcp.((system_domain))
 
+## APISERVER - remove publicapiserver
 - type: remove
-  path: /instance_groups/name=metricsforwarder
-
-
-## SCALINGENGINE - Enable cf Server to receive calls from api running on cf --
-- type: replace
-  path: /instance_groups/name=scalingengine/jobs/name=scalingengine/properties/autoscaler/scalingengine/cf_server?/xfcc?/valid_org_guid?
-  value: ((autoscaler_cf_server_xfcc_valid_org_guid))
-
-- type: replace
-  path: /instance_groups/name=scalingengine/jobs/name=scalingengine/properties/autoscaler/scalingengine/cf_server?/xfcc?/valid_space_guid?
-  value: ((autoscaler_cf_server_xfcc_valid_space_guid))
-
-
-- type: replace
-  path: /instance_groups/name=scalingengine/jobs/name=scalingengine/properties/autoscaler/scalingengine/cf_server?/port?
-  value: &scalingEngineCfPort 6205
+  path: /instance_groups/name=apiserver
 
-- type: replace
-  path: /instance_groups/name=postgres/jobs/name=route_registrar/properties/route_registrar/routes/-
-  value:
-    name: ((deployment_name))-cf-scalingengine
-    registration_interval: 20s
-    port: *scalingEngineCfPort
-    tags:
-      component: autoscaler_cf_scalingengine
-    uris:
-      - ((deployment_name))-cf-scalingengine.((system_domain))
-
-## EVENTGENERATOR - Enable cf Server to receive calls from api running on cf --
-
-- type: replace
-  path: /instance_groups/name=eventgenerator/jobs/name=eventgenerator/properties/autoscaler/eventgenerator/cf_server?/xfcc?/valid_org_guid?
-  value: ((autoscaler_cf_server_xfcc_valid_org_guid))
-
-- type: replace
-  path: /instance_groups/name=eventgenerator/jobs/name=eventgenerator/properties/autoscaler/eventgenerator/cf_server?/xfcc?/valid_space_guid?
-  value: ((autoscaler_cf_server_xfcc_valid_space_guid))
 
```

Comment:
Since the reason to change is CF Apps and we don't need the vms, I would move the removal of the VMs in the operations/configure-cf-services.yml file.